src/components/Send.css

```diff
@@ -122,3 +122,15 @@ input::-webkit-inner-spin-button {
   background-color: var(--bs-gray-dark);
   border-color: var(--bs-gray-dark) !important;
 }
+
+.input-loader {
+  height: 3.5rem;
+}
```

Comment:
```suggestion
.send .input-loader {
  height: 3.5rem;
}
```

Since this file is not a CSS Module, we have to be careful about naming clashes.

Reply:
Done. I've moved all css that isn't global into an appropriate module.